source/shared/smart-contracts/guides/setup-tools.rst

```diff
@@ -32,7 +32,14 @@ It can be used for :ref:`compiling<compile-module>` and
    Add links for testing and schemas.
 
 Cargo Concordium is distributed as part of the :ref:`Concordium software<downloads>` package.
-The tool should be placed in your PATH.
+The tool should be placed in your PATH. This can be achieved by either:
+
+* Moving the executable to a folder that is already on your PATH, for example ``C:\Users\<your-username>\.cargo\bin\`` (replacing <your-username> with your actual username).
```

Comment:
```suggestion
* Moving the executable to a folder that is already on your PATH, for example ``%HOMEPATH%\.cargo\bin\`` for Windows, and ``$HOME/.cargo/bin`` for MacOS/Linux
```

The folder path is only valid for windows. I guess it might be fine as it is an example as to where it could be placed. Also I think it might be nice to use the home directory variables instead of username replacement.